DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/board_test/src/board_test.cpp

```diff
@@ -49,9 +49,11 @@ int main(int argc, char* argv[]) {
 //  - the FPGA emulator device (CPU emulation of the FPGA) using FPGA_EMULATOR
 //  macro
 //  - the FPGA device (a real FPGA)
-#if FPGA_HARDWARE
+#if FPGA_SIMULATOR
+    auto selector = sycl::ext::intel::fpga_simulator_selector_v;
```

Comment:
The sim takes too long for this design, so it is voluntarily disabled.

Reply:
Ah okay -- I had added this and meant to ask someone regarding sim support but forgot. I'll remove it from the sample

Reply:
I've removed the sim selector from the source code. i haven't removed the sim flow from the CMake yet though -- do you think we should? I know Paul had wanted us to use a common CMake between samples. the sim flow is not mentioned in the README

Reply:
This is fine as it is